tests/functional/tests/conftest.py

```diff
@@ -44,6 +44,15 @@ def pyocf_ctx_log_buffer():
     gc.collect()
 
 
+def pytest_addoption(parser):
+    parser.addoption("--debug-log", action="store_true", help="enable debug logs")
+
+
+@pytest.fixture
+def debug_log(request):
+    return request.config.getoption("--debug-log")
+
+  
```

Comment:
Pytest already has it's own log level setting. Any chance to use it instead of introducing additional adoption?